react/features/base/participants/reducer.js

```diff
@@ -238,15 +230,11 @@ ReducerRegistry.register('features/base/participants', (state = DEFAULT_STATE, a
 
         state.remote.set(id, participant);
 
-        // Insert the new participant.
-        const displayName = _getDisplayName(name);
-        const sortedRemoteParticipants = Array.from(state.sortedRemoteParticipants);
+        // Insert the new participant and sort it alphabetically.
+        const remoteParticipants = Array.from(state.sortedRemoteParticipants.keys());
 
-        sortedRemoteParticipants.push([ id, displayName ]);
-        sortedRemoteParticipants.sort((a, b) => a[1].localeCompare(b[1]));
-
-        // The sort order of participants is preserved since Map remembers the original insertion order of the keys.
-        state.sortedRemoteParticipants = new Map(sortedRemoteParticipants);
+        remoteParticipants.push(id);
+        state.sortedRemoteParticipants = _sortParticipantsByDisplayname(state, remoteParticipants);
```

Comment:
I find the new version less efficient than the previous one because we loop trough the remote participants one additional time. Is this correct? Why did you change it?